src/main/kotlin/string/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+package string
+
+class Calculator() {
```

Comment:
class가 상태 값을 가지지 않아서 싱글턴 객체인 object로 변경해도 좋다는 생각이 드네요! :)
```suggestion

object Calculator {
```